SpywareFilter/sections/specific.txt

```diff
@@ -23,6 +23,26 @@
 ||adc.nine.com.au/?token=$redirect=nooptext,important,domain=9now.com.au
 ||googleadservices.com/pagead/conversion_async.js$redirect=noopjs,domain=ffm.to,important
 !
+! realty.chosun.com - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||news.chosun.com/svc/fb_count.html
+||news.chosun.com/dhtm/*/collecter.js
+! w.atwiki.jp - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||w.atwiki.jp/*/google_analytics.js
+||w.atwiki.jp/*/counter.js
+! ytn.co.kr - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||ats.auction.co.kr/cpc/imp/nw?r
+||engine.tend-table.com/cgi-bin/WebLog.dll
+||grenade.ebaykorea.com/api/getImps
+||openapi.tv.zum.com/logs/ad/
+! theqoo.net - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||cdn.theqoo.net/counter.js
+! clien.net - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||clien.net/service/*/clien.count.js
+! ruliweb.com - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||image.ruliweb.com/view/log.htm
+! gall.dcinside.com - https://github.com/AdguardTeam/AdguardFilters/pull/75313
+||d-api.jennifersoft.com/api/collect-log
```

Comment:
Cannot open
![image](https://user-images.githubusercontent.com/8361299/109548188-0080dd00-7ad5-11eb-87fe-a0d44d7b051b.png)
It seems that service is for developers
http://cdn.jennifersoft.com/wp-content/uploads/Documents/en/JENNIFER_White_Paper-Performance_Analysis_Model.pdf

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/75313/commits/b77b7616a50c637661f561a74c6208eeffc7d547